pulp_ansible/app/tasks/collections.py

```diff
@@ -324,7 +324,7 @@ def _get_url(page, api_version):
                 name, version, source = collection_info[page - 1]
                 namespace, name = name.split(".")
                 root = source or remote.url
-                url = f"{root}/api/v2/collections/{namespace}/{name}"
+                url = f"{root}/{namespace}/{name}"
```

Comment:
it picks source or the remote.url, so I had to validate it

Reply:
I see how this correctly handles the case of forming the url for the requirements file, but in the case of no requirements file, where does it form those urls from?

Reply:
if it does not have a requirements file, it does not fall on this if

Reply:
Cool, so to confirm, we are able to form the right urls when not using a requirements file?

Reply:
yes, we are, we have lots of tests doing that